tests/doghouse/test_repo_context.py

```
@@ -0,0 +1,64 @@
+def test_resolve_explicit_repo_and_pr():
+    """When both --repo and --pr are provided, no auto-detection needed."""
+    repo, owner, name, pr = resolve_repo_context("flyingrobots/draft-punks", 42)
+    assert repo == "flyingrobots/draft-punks"
+    assert owner == "flyingrobots"
+    assert name == "draft-punks"
+    assert pr == 42
+
+
+def test_resolve_parses_owner_name_from_repo_string():
+    """The repo string should be split into owner and name."""
+    repo, owner, name, pr = resolve_repo_context("acme/widgets", 7)
```

Ruff (0.15.7)

[warning] Unpacked variable `repo` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)